runtime-builder/tests/integration/test.sh

```diff
@@ -17,7 +17,7 @@
 # test.sh deploys the test app and run the integration test on it.
 # usage: test.sh <project_id>
 
-usage() { echo "Usage: $0 <project_id>"; exit 1; }
+usage() { echo "Usage: $0 <project_id> <builder_image_tag>"; exit 1; }
```

Comment:
[builder_image_tag] instead of <builder_image_tag> to indicate optional parameter.